fonts/Makefile.am

```diff
@@ -81,6 +81,25 @@ dist_font_DATA = \
 	Merriweather-Regular.ttf \
 	Merriweather-UltraBold.ttf \
 	Merriweather-UltraBdIt.ttf \
+	Metropolis-BlackItalic.otf \
+	Metropolis-Black.otf \
+	Metropolis-BoldItalic.otf \
+	Metropolis-Bold.otf \
+	Metropolis-ExtraBoldItalic.otf \
+	Metropolis-ExtraBold.otf \
+	Metropolis-ExtraLightItalic.otf \
+	Metropolis-ExtraLight.otf \
+	Metropolis-LightItalic.otf \
+	Metropolis-Light.otf \
+	Metropolis-MediumItalic.otf \
+	Metropolis-Medium.otf \
+	Metropolis-OFL.md \
```

Comment:
I don't think this file is a font so it shouldn't be here. I couldn't find it either in the list of files you added.

Reply:
Good catch. I removed that line locally but forgot to `git --amend` apparently